src/components/sidebar/CallTreeSidebar.js

```diff
@@ -80,7 +82,8 @@ class ImplementationBreakdown extends React.PureComponent<
   }
 
   render() {
-    const { breakdown } = this.props;
+    const { breakdown, isIntervalInteger } = this.props;
```

Comment:
Hey @monikamaheshwari, the Flow check fails here because `isIntervalInteger` should be added to the props type definition for this component, this is line 51. You can simply add a line like this:
```js
type ImplementationBreakdownProps = {|
   +breakdown: BreakdownByImplementation,
   +isIntervalInteger: boolean,
|};
```

Thanks again :)